### PR TITLE
Amend/responsive manage ui

### LINKED ISSUE #116: Responsive design: Manage UI

### PR DESCRIPTION
**Pull request by M4R1KU**
This closes #116  #166 #161.

Summary of changes:
- ​added responsive design for manage-ui
- made general changes to make the whole site more responsive

I made sure to:
- [x] Add and / or update tests
- [x] Run the code formatter (`CTRL + ALT + L`)
- [x] Update any documentation (javadoc, domain model, erd, ...)
- [x] Follow the contribution guidelines
- [x] Update CI scripts on Semaphore (changes to `auth0.yml` or `application.yml`)

Please review @jmesserli @needToRoll  / cc: @outcobra/developers.

### CODE REVIEW

**Comment by M4R1KU** [comment]
Please review as soon as possible

**Comment by jmesserli** [comment]
Did you fix the problem we saw yesterday?

Mario Kunz <notifications@github.com> schrieb am Di., 14. März 2017, 18:04:

**Comment by M4R1KU** [comment]
The one with the config?

**Comment by M4R1KU** [comment]
I did not fix the problem bc I don't now a good solution. 
But I created an issue on our project and commented on an another issue which describes a very similar problem on the angular repository.
I hope that the finally fix this bc it's not really a problem in our code.

If they don't fix until our first beta stage we should find a temporary alternative to our current approach.

My issue #158

**Comment by jmesserli** [comment]
Nope i mean the one with the button that was unreachable on mobile

**Comment by M4R1KU** [comment]
Can you reproduce it and create an issue? Then I won't forget about it again.

**Comment by jmesserli** [comment]
I'll try

Mario Kunz <notifications@github.com> schrieb am Di., 14. März 2017, 20:35:

**Comment by jmesserli** [comment]
Is outcobra-a the newest version? @M4R1KU

File: frontend/src/app/manage/manage.component.ts [hunk 1]
```diff
@@ -52,14 +65,80 @@ export class ManageComponent implements OnInit {
...
+
+    private setColumnClasses() {
+        this.columnClasses = {
+            'col': !this.isMobile(),
+            's4': !this.isMobile(),
+            'mobile-col': this.isMobile()
+        };
+    }
+
+    private changeView(next: number) {
```
Comment (jmesserli) [inline frontend/src/app/manage/manage.component.ts]: Why can this not be removed?

File: frontend/src/app/shared/services/auth/auth.service.ts [hunk 2]
```diff
@@ -16,7 +18,12 @@ export class AuthService {
     private redirectRoute: string;
     lock;
 
-    constructor(private config: Config, private router: Router, private http: HttpInterceptor, private notificationService: NotificationsService) {
+    constructor(private config: Config,
+                private router: Router,
```
Comment (jmesserli) [inline frontend/src/app/shared/services/auth/auth.service.ts]: I think your codestyle is wrongly set up to use `"`

**Comment by jmesserli** [comment]
Looks good generally, there are some things open from [OC-CR-629](https://upsource.pegnu.cloud/outstanding-cobra/review/OC-CR-629)

**Pull request merged**